htmresearch/frameworks/classification/utils/traces.py

```diff
@@ -165,6 +165,9 @@ def loadTraces(fileName):
                             'sensorValue',
                             'actualCategory',
                             'predictedCategory',
+                            'anomalyScore',
+                            'clusteringConfidence',
+                            'predictedClusterLabel',
                             'predictedCategoryTrace']:
```

Comment:
Correct. Another good catch. I'll remove it `predictedCategoryTrace `, it's unused.